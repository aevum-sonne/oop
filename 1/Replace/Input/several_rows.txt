The Smiths signed to the independent record label Rough Trade Records. They have also released several compilations and non-album singles. Several Smiths singles reached the top 20 of the UK Singles Chart and all of their studio albums reached the top five of the UK Albums Chart, including the number-one album Meat Is Murder (1985).

