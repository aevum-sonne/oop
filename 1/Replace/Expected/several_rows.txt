The Smiths signed to 123 independent record label Rough Trade Records. They have also released several compilations and non-album singles. Several Smiths singles reached 123 top 20 of 123 UK Singles Chart and all of 123ir studio albums reached 123 top five of 123 UK Albums Chart, including 123 number-one album Meat Is Murder (1985).

